dask/dataframe/tests/test_shuffle.py

```diff
@@ -916,3 +916,47 @@ def test_set_index_timestamp():
 
     assert_eq(df2, ddf_new_div)
     assert_eq(df2, ddf.set_index("A"))
+
+
+@pytest.mark.parametrize("compression", [None, "ZLib"])
+def test_disk_shuffle_with_compression_option(compression):
+    # test if dataframe shuffle works both with and without compression
+    with dask.config.set({"dataframe.shuffle-compression": compression}):
+        test_shuffle("disk")
+
+
+@pytest.mark.parametrize("compression", ["UNKOWN_COMPRESSION_ALGO"])
+def test_disk_shuffle_with_unknown_compression(compression):
+    # test if dask raises an error in case of fault config string
+    with dask.config.set({"dataframe.shuffle-compression": compression}):
+        with pytest.raises(ImportError) as excinfo:
```

Comment:
You can use `match="load {}".format(compression)` rather than excinfo and the assert.